components/segmenter_lstm/tests/testdata/test_text_codepoints.json

```diff
@@ -1,3 +1,32 @@
+                {"unseg": "เพียงแค่สะกิดเตือนให้ก้มลงกราบลาอีกครั้งก่อนที่จะถอยนำออกมาจาก", "expected_bies": "biiiebiebiiiebiiiebiebiebiiiiiiebiebiiiebiiebiebebiebebiebebie", "true_bies": "biiiebiebiiiebiiiebiebiebebiiebebiebiiiebiiebiebebiebebiebebie"},
+                {"unseg": "ค่ะ", "expected_bies": "bie", "true_bies": "bie"},
+                {"unseg": "แม่ตอบอย่างมี", "expected_bies": "biebiebiiiebe", "true_bies": "biebiebiiiebe"},
+                {"unseg": "หนังสือที่พิมพ์ออกมาทุกวันจะมีชื่อหนังสือชื่อเดียวกันตลอด", "expected_bies": "biiiiiebiebiiiebiebebiebiebebebiiebiiiiiebiiebiiiebiebiii", "true_bies": "biiiiiebiebiiiebiebebiebiebebebiiebiiiiiebiiebiiiebiebiie"},
+                {"unseg": "หรือหนังสือที่พิมพ์ออกมาทุกสัปดาห์", "expected_bies": "biiebiiiiiebiebiiiebiebebiebiiiiie", "true_bies": "biiebiiiiiebiebiiiebiebebiebiiiiie"},
+                {"unseg": "ทุกสองสัปดาห์", "expected_bies": "biebiebiiiiie", "true_bies": "biebiebiiiiie"},
+                {"unseg": "พับเป็นเล่มโดยไม่เย็บเล่มและไม่มีปก", "expected_bies": "biebiiebiiebiebiebiiebiiebiebiebebi", "true_bies": "biebiiebiiebiebiebiiebiiebiebiebebe"},
+                {"unseg": "เป็นสำคัญ", "expected_bies": "biiebiiie", "true_bies": "biiebiiie"},
+                {"unseg": "การเรียงพิมพ์จะเรียงเป็นคอลัมน์เพราะสะดวกในการนำมาจัดหน้า", "expected_bies": "biebiiieiiiiebebiiiebiiebiiiiiebiiiebiiiebebiebebebiebiii", "true_bies": "biebiiiebiiiebebiiiebiiebiiiiiebiiiebiiiebebiebebebiebiie"},
+                {"unseg": "จึงควรเลือกข่าวสำคัญๆหลายข่าวลงพิมพ์รวมไว้ในหน้าแรก", "expected_bies": "biebiebiiiebiiebiiiesbiiebiiebebiiiebiebiebebiiebie", "true_bies": "biebiebiiiebiiebiiiesbiiebiiebebiiiebiebiebebiiebie"},
+                {"unseg": "ทุกสถานที่", "expected_bies": "biebiiiiie", "true_bies": "biebiiiiie"},
+                {"unseg": "ไม่ว่าจะกำลังดื่มกาแฟ", "expected_bies": "bieiiebebiiiebiiebiie", "true_bies": "biiiiebebiiiebiiebiie"},
+                {"unseg": "โดยสารรถหรือเรือ", "expected_bies": "biebiebebiiebiie", "true_bies": "biiiiebebiiebiie"},
+                {"unseg": "แม้มีเวลาอ่านเพียงเล็กน้อยถ้ามีข่าวที่ตนสนใจก็จะอ่านก่อน", "expected_bies": "iiebebiiebiiebiiiebiiiiiiebiebebiiebiebebiiebebebiiebiie", "true_bies": "biebebiiebiiebiiiebiiiiiiebiebebiiebiebebiiebebebiiebiie"},
+                {"unseg": "ประกอบด้วยข่าวหลายข่าว", "expected_bies": "biiiiebiiebiiebiiebiie", "true_bies": "biiiiebiiebiiebiiebiie"},
+                {"unseg": "จึงมีราคาอยู่เพียงวันเดียวเมื่อพ้นวันไปแล้วก็หมดราคา", "expected_bies": "biebebiiebiiebiiiebiebiiiebiiiebiebiebebiiebebiebiii", "true_bies": "biebebiiebiiebiiiebiebiiiebiiiebiebiebebiiebebiebiie"},
+                {"unseg": "นิตยสาร", "expected_bies": "biiebii", "true_bies": "biiiiie"},
+                {"unseg": "ทางด้านความคิด", "expected_bies": "biebiiebiiebii", "true_bies": "iiibiiebiiebie"},
```

Comment:
Why would true bies start with i?

Reply:
Nice observation! I think it was because in copy-pasting I missed the first pipe and `ทาง|ด้าน|ความ|คิด|` was copied instead of `|ทาง|ด้าน|ความ|คิด|`. It didn't affect the estimated BIES because they have the same unsegmented version, so I didn't realize it in my `assert_eq!`. Fixed it.